package/run.ps1

```diff
@@ -48,12 +48,13 @@ function Get-StringHash {
 
 $rke2ServiceName = "rke2"
 $SA_INSTALL_PREFIX = "c:/usr/local"
-$SAI_FILE_DIR = "c:/var/lib/rancher/rke2/system-agent-installer"
+$RKE2_DATA_DIR = if ($env:RKE2_DATA_DIR) { $env:RKE2_DATA_DIR } else { "c:/var/lib/rancher/rke2" };
+$SAI_FILE_DIR = "$RKE2_DATA_DIR/system-agent-installer"
 $RESTART_STAMP_FILE = "$SAI_FILE_DIR/rke2_restart_stamp"
 $PRIOR_RESTART_STAMP = ""
 $RESTART = $false
 
-New-Directroy -Path "c:/var/lib/rancher/rke2"
+New-Directroy -Path "$RKE2_DATA_DIR"
 New-Directroy $SAI_FILE_DIR
```

Comment:
I am guessing that this is currently doing nothing because of the typo. Should be `New-Directory` not New-Directroy`

Reply:
No idea why but a `New-Directroy` is defined on line 22, which defines the `-Path` parameter. I'm guessing it's to better handle the case where a directory already exists? 

https://github.com/rancher/system-agent-installer-rke2/blob/8d902f75a75ebb3d9059af4d02923dbad44927e6/package/run.ps1#L22-L32

Reply:
Oh, more fun stuff left over from Jamie and the rush-job that was initial Windows support 😬 . Can we fix the typo and inconsistent use of -Path while we're here?